compiler/rustc_codegen_llvm/src/gotoc/mir_to_goto/utils/utils.rs

```diff
@@ -1,15 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-use super::cbmc::goto_program::{Expr, Location, Stmt, SymbolTable, Type};
-use super::metadata::*;
 use crate::btree_string_map;
-use rustc_hir::def_id::DefId;
-use rustc_middle::mir::{Local, VarDebugInfo, VarDebugInfoContents};
-use rustc_middle::ty::print::with_no_trimmed_paths;
-use rustc_middle::ty::Instance;
-use rustc_span::Span;
-use tracing::debug;
+use crate::gotoc::cbmc::goto_program::{Expr, Location, Stmt, SymbolTable, Type};
+use crate::gotoc::mir_to_goto::GotocCtx;
 
+// Should move into rvalue
+//make this a member function
```

Comment:
Should this have an issue?

Reply:
https://github.com/model-checking/rmc/issues/427